scripts/cdn_publish.sh

```diff
@@ -16,23 +20,34 @@ then
   version=`echo $allVersions | cut -d" " -f1`
   # be sure we get only the version part and discard any prefix
   version=`expr $version : '.*\(v[0-9].*\)'`
-
-  mkdir -p ${cdnDir}/${cdnGravityDir}/${version}
-
-  cp packages/gravity-ui-web/dist/ui-lib/* ${cdnDir}/${cdnGravityDir}/${version}
+  
+  mkdir -p ${cdnCurrentVersionGravityDir}/${version}
+  cp packages/gravity-ui-web/dist/ui-lib/* ${cdnCurrentVersionGravityDir}/${version}
 
   versionList=""
   for thisVersion in $allVersions; do
     versionList+="<li>`expr $thisVersion : '.*\(v[0-9].*\)'`</li>"
   done
 
   file_contents=$(<./cdn-index.template)
-  echo "${file_contents//\{versionlist\}/$versionList}" > ${cdnDir}/${cdnGravityDir}/index.html
+  echo "${file_contents//\{versionlist\}/$versionList}" > ${cdnCurrentVersionGravityDir}/index.html
+
+  ~/.local/bin/aws s3 sync ${cdnCurrentVersionDir} s3://${CDN_BUCKET} --region=${PROD_BUCKET_REGION} --exclude "*" --include "*.html" --content-type "text/html; charset=utf-8" --cache-control "max-age=0"
+  ~/.local/bin/aws s3 sync ${cdnCurrentVersionDir} s3://${CDN_BUCKET} --region=${PROD_BUCKET_REGION} --exclude "*" --include "*.css" --content-type "text/css; charset=utf-8" --cache-control "max-age=3153600000, immutable" 
+  ~/.local/bin/aws s3 sync ${cdnCurrentVersionDir} s3://${CDN_BUCKET} --region=${PROD_BUCKET_REGION} --exclude "*" --include "*.js" --content-type "application/javascript; charset=utf-8" --cache-control "max-age=3153600000, immutable"
+  ~/.local/bin/aws s3 sync ${cdnCurrentVersionDir} s3://${CDN_BUCKET} --region=${PROD_BUCKET_REGION} --include "*" --exclude "*.js" --exclude "*.html" --exclude "*.css" --cache-control "max-age=3153600000, immutable" 
+
+  majorVersion=`echo $version | cut -dv -f2 | cut -d. -f1`
+
+  mkdir -p ${cdnLatestVersionsGravityDir}/${majorVersion}.x
```

Comment:
Can we change the path here and n the next line to add the `v` prefix back. I think it'd be neater if each latest major version directory was along the lines of `v2.x/` rather than `2.x/`.

That way, if someone was looking at a directory listing of all release folders, those ones will appear alongside the individual `v2.x.y` release folders